src/components/item/settings/ThumbnailSetting.tsx

```diff
@@ -109,8 +114,9 @@ const ThumbnailSetting = ({ item }: Props): JSX.Element | null => {
           alt={alt}
           url={
             imgUrl ??
-            (item.type === ItemType.LINK ? item.extra[ItemType.LINK] : {})
-              ?.thumbnails?.[0]
+            (item.type === ItemType.LINK
+              ? getLinkThumbnailUrl(item.extra)
```

Comment:
I'm not sure to understand here. If `imgUrl` is undefined, `getLinkThumbnailUrl(item.extra)` will return undefined no ? Because `imgUrl` will be set by `getLinkThumbnailUrl`, or am I missing something ?